frontend/src/components/addOrder/OrderEntryAdditionalQuestions.js

```diff
@@ -48,9 +49,9 @@ export const ProgramSelect = ({
 
   return (
     <>
-      <div className="formInlineDiv">
-        {programs.length > 0 && (
-          <div className="inputText">
+      <Grid fullWidth={true}>
```

Comment:
The layout is also broken on Edit order but works fine on Add Odrer

![Uploading image.png…]()